tests/opentelemetry-docker-tests/tests/redis/test_redis_functional.py

```diff
@@ -90,6 +149,27 @@ def test_pipeline_traced(self):
         self.assertEqual(span.attributes.get("db.redis.pipeline_length"), 3)
 
     def test_pipeline_immediate(self):
+        with self.redis_client.pipeline() as pipeline:
+            pipeline.set("a", 1)
+            pipeline.immediate_execute_command("SET", "b", 2)
+            pipeline.execute()
+
+        spans = self.memory_exporter.get_finished_spans()
```

Comment:
I suggest to keep the old function tests and just move sanitize_query=False to them
so the test without sanitizing will not change.
and add new function to check sanitizing like you did.
BTW there is an open issue in the spesification if the default db.statement sanitization should be True/False 
https://github.com/open-telemetry/opentelemetry-specification/issues/3104